pycbc/io/gracedb.py

```diff
@@ -136,7 +136,7 @@ def __init__(self, coinc_ifos, ifos, coinc_results, **kwargs):
         coinc_event_row.time_slide_id = lsctables.TimeSlideID(0)
         coinc_event_row.process_id = proc_id
         coinc_event_row.coinc_event_id = coinc_id
-        coinc_event_row.likelihood = 0.
+        coinc_event_row.likelihood = coinc_results['foreground/stat']
```

Comment:
Will this also work for offline uploads and for `pycbc_make_skymap`, or do we need to check if `coinc_results` has that key?

Reply:
For offline uploads, it looks like they use the `to_coinc_xml_object` function from `pycbc/io/hdf.py`, which already does this

`pycbc_make_skymap` doesn't provide this, so it should be checked, Ive added that now